controller/main.go

```diff
@@ -147,6 +148,11 @@ func (c *controller) SetPools(l log.Logger, pools *config.Pools) controllers.Syn
 	return controllers.SyncStateReprocessAll
 }
 
+func (c *controller) SetNode(l log.Logger, node *v1.Node) controllers.SyncState {
+	c.nodes[node.Name] = node
```

Comment:
nit: although I am asking to remove this, writing it to let you know that we should be careful to consider that this map is not aligned with the content of the informer cache, and only expands. If a node gets removed, its ips will still be not valid.

Reply:
I see. As long as the removed nodes' IP aren't made available in the pool for the services, there won't be any risk of conflict.

Reply:
Comment is resolved. Code is removed.